assets/sass/v2/_password.scss

```diff
@@ -0,0 +1,11 @@
+.password-authenticator {
+  &--heading {
+    margin-bottom: 1rem;
```

Comment:
BTW the original thread got converted to a discussion on class names. I was asking about the value of `1rem`.

> Maybe not for this PR, but are we using Odyssey standards for v2? If so, can we use standard spacing units from Odyssey? If not, might be worth it to consolidate the spacing units in one file and then re-use. Eventually, we can convert from that one place to design systems when we move.

So, the idea is that we create variables like `$base-spacing: 1rem;` and re-use that everywhere rather than having different spacing units in different places.

Reply:
Sure, I think it makes sense to address it as a overall refactor and possibly fetch odyssey constants + add more if necessary.. We will need to discuss on what path we take and work on it as a separate PR.

Reply:
Yeah, that sounds good to me! Might be a good widget meeting topic as well.